jobs/payment-jobs/tasks/ejv_payment_task.py

```diff
@@ -0,0 +1,222 @@
+                        line_number += 1
+                        control_total += 1
+                        account_jv = account_jv + cls.get_jv_line(batch_type, debit_distribution, disbursement_desc,
+                                                                  effective_date, flow_through, journal_name,
+                                                                  line.total,
+                                                                  line_number, 'D')
+                    if line.service_fees > 0:
+                        total += line.service_fees
+                        service_fee_distribution = cls.get_distribution_string(service_fee_distribution_code)
+                        flow_through = f'{line.invoice_id:<110}'
+                        # Credit to BCREG GL
+                        line_number += 1
+                        control_total += 1
+                        account_jv = account_jv + cls.get_jv_line(batch_type, service_fee_distribution,
+                                                                  disbursement_desc,
+                                                                  effective_date, flow_through, journal_name,
+                                                                  line.service_fees,
+                                                                  line_number, 'C')
+
+                        # Debit from GOV ACCOUNT GL
+                        line_number += 1
+                        control_total += 1
+                        account_jv = account_jv + cls.get_jv_line(batch_type, debit_distribution, disbursement_desc,
+                                                                  effective_date, flow_through, journal_name,
+                                                                  line.service_fees,
+                                                                  line_number, 'D')
+            batch_total += total
+
+            # A JV header for each account.
+            account_jv = cls.get_jv_header(batch_type, cls.get_journal_batch_name(batch_number),
+                                           journal_name, total) + account_jv
+            ejv_content = ejv_content + account_jv
+
+            # Create ejv invoice link records and set invoice status
+            for inv in invoices:
+                # Create Ejv file link and flush
+                EjvInvoiceLinkModel(invoice_id=inv.id, ejv_header_id=ejv_header_model.id,
+                                    disbursement_status_code=DisbursementStatus.UPLOADED.value).flush()
+                # Set distribution status to invoice
+                # Create invoice reference record
+                inv_ref = InvoiceReferenceModel(
+                    invoice_id=inv.id,
+                    invoice_number=generate_transaction_number(inv.id),
+                    reference_number=None,
+                    status_code=InvoiceReferenceStatus.ACTIVE.value
+                )
+                inv_ref.save()
```

Comment:
do you want to save or flush here?

Reply:
Yes, should be flush. Will fix this in next PR as I am already changing this file for reconciliation